data/avie-dev.json

```diff
@@ -1,8 +1,15 @@
 {
   "name": "Avie Fukai",
   "type": "personal",
+  "displayStatsPublic": true,
```

Comment:
This is no longer needed, as you can login to see your stats

Reply:
```suggestion
```

Reply:
Thank you Eddie!